CoreDataAccessView/test/unit/src/au/gov/asd/tac/constellation/views/dataaccess/panes/DataAccessSearchProviderNGTest.java

```diff
@@ -0,0 +1,324 @@
+import org.netbeans.spi.quicksearch.SearchResponse;
+import org.openide.awt.Notification;
+import org.openide.awt.NotificationDisplayer;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+/**
+ *
+ * @author aldebaran30701
+ */
+public class DataAccessSearchProviderNGTest {
+
+    @Mock
+    SearchResponse response;
+
+    @Mock
+    SearchRequest request;
+
+    @Mock
+    DataAccessPane daPane;
+
+    @Mock
+    QueryPhasePane qpPane;
+
+    protected NotificationDisplayer notifDisplayer;
+
+    public DataAccessSearchProviderNGTest() {
+    }
+
+    @BeforeClass
+    public void setUpClass() throws Exception {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @AfterClass
+    public static void tearDownClass() throws Exception {
+    }
+
+    @BeforeMethod
+    public void setUpMethod() throws Exception {
+    }
+
+    @AfterMethod
+    public void tearDownMethod() throws Exception {
+        // Check usage is correct before moving on
+        validateMockitoUsage();
+    }
+
+    /**
+     * Test of evaluate method, of class DataAccessSearchProvider. Pass in an
+     * unmatchable search string
+     */
+    @Test
+    public void testEvaluateFail() {
+        System.out.println("evaluate fail");
+
+        // Creating mocks
+        MockitoAnnotations.initMocks(this);
+        daPane = mock(DataAccessPane.class);
+        qpPane = mock(QueryPhasePane.class);
+        request = mock(SearchRequest.class);
+        response = mock(SearchResponse.class);
+
+        // Create mock of DataAccessPane to return the query phase pane mock
+        try (MockedStatic<DataAccessPane> mockedStatic = Mockito.mockStatic(DataAccessPane.class)) {
+            mockedStatic.when(() -> DataAccessPane.getQueryPhasePane(Mockito.any())).thenReturn(qpPane);
+        }
+
+        // Create mock of DataAccessUtilities to return the data access pane mock
+        // Mock of DataAccessPane will return a blank new tab when called
+        try (MockedStatic<DataAccessUtilities> mockedStatic2 = Mockito.mockStatic(DataAccessUtilities.class)) {
+            when(daPane.getCurrentTab()).thenReturn(new Tab());
+            mockedStatic2.when(() -> DataAccessUtilities.getDataAccessPane()).thenReturn(daPane);
+        }
+
+        // Mock the request text to be an unmatchable string
+        when(request.getText()).thenReturn("nothignshouldmatchthisstring");
+        when(response.addResult(Mockito.anyObject(), Mockito.anyString())).thenReturn(true);
+        DataAccessSearchProvider instance = new DataAccessSearchProvider();
+        instance.evaluate(request, response);
+
+        // Verify that addResult was never called.
+        // This should mean that no plugin name matched the input
+        verify(response, never()).addResult(Mockito.anyObject(), Mockito.anyString());
```

Comment:
`anyObject()` is going to be deprecated in Mockito 4. Can we use `new Object()` or something like this to avoid this becoming a problem in the future upgrades given at the time of deving this we know it is marked for deprecation. See https://www.javadoc.io/static/org.mockito/mockito-core/3.11.2/org/mockito/ArgumentMatchers.html#anyObject-- for more information.

Reply:
any() seems to be the next in line to use, I originally had it as any() but I was having issues elsewhere and was changing things here to check if they would help me.